packages/react-query/src/scripts/utils/commands.ts

```diff
@@ -3,6 +3,19 @@ import path from 'path'
 import { getPackageJson, getTanStackReactQueryPackageJson } from './package'
 import { switchVersion } from './switchVersion'
 
+export const version5APIs = ['SuspenseQuery', 'SuspenseQueries', 'SuspenseInfiniteQuery', 'QueryErrorBoundary']
```

Comment:
I have temporarily declared the API list. In the next version, I will parse the TypeScript to list and display the APIs!

Reply:
cool 👍 Fast resolve issue